src/base/AsyncJob.cc

```diff
@@ -179,3 +188,26 @@ const char *AsyncJob::status() const
     return buf.content();
 }
 
+void
+AsyncJob::ReportAllJobs(StoreEntry *e)
+{
+    PackableStream os(*e);
+    // this loop uses YAML syntax, but AsyncJob::status() still needs to be adjusted to use YAML
+    const char *indent = "    ";
+    for (const auto job: Jobs_) {
+        os << indent << job->id << ":\n";
+        os << indent << indent << "type: '" << job->typeName << "'\n";
+        os << indent << indent << "status:" << job->status() << '\n';
```

Comment:
YAML "flow sequence" should be wrapped in square brackets:
```suggestion
        os << indent << indent << "status: [" << job->status() << "]\n";
```